source/memory-store.ts

```diff
@@ -102,9 +117,9 @@ export default class MemoryStore implements Store {
 	 * @public
 	 */
 	async decrement(key: string): Promise<void> {
-		const current = this.hits[key]
+		const client = this.getClient(key)
 
-		if (current) this.hits[key] = current - 1
+		if (client.totalHits > 1) client.totalHits--
```

Comment:
Shouldn't this be `> 0`?

Reply:
No, because we don't want it to ever go negative.

Reply:
But if the number of hits is 1 it should be able to decrement it to 0 right?

Reply:
🤦 yes.